pod-operation/src/components/inverter_board.rs

```diff
@@ -0,0 +1,18 @@
+use rppal::uart::{Parity, Uart};
+pub struct InverterBoard {
+	uart: Uart,
+}
+
+impl InverterBoard {
+	pub fn new() -> Self {
+		let uart = Uart::with_path("/dev/ttyACM0", 9600, Parity::None, 8, 1).unwrap();
+		Self { uart }
+	}
+
+	pub fn send_control(&mut self, velocity: f32, throttle: f32) {
+		let vel_str = velocity.to_string();
+		let throttle_str = throttle.to_string();
+		let data = vel_str + " " + &throttle_str + "\n";
```

Comment:
Thought: I think it might be simpler to construct the message string using the [`format!`](https://doc.rust-lang.org/std/macro.format.html) macro rather than concatenating.